src/Snackbar/Snackbar.react.js

```diff
@@ -86,24 +90,27 @@ class Snackbar extends PureComponent {
         const styles = getStyles(props, context);
         this.state = {
             styles,
-            moveAnimated: new Animated.Value(0),
+            moveAnimated: new Animated.Value(48),
```

Comment:
What's the 48? It should have been height from styles, right?

It should be set to 0 if visible === false, or maybe -height?